module-types/Datacom/PSU-120-DC_OLT-DM4610.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 120 DC
```

Comment:
Is this the actual item name? If so, that is fine. Otherwise please modify this to match the actual name of the item.

Reply:
According to [this datasheet](https://www.datacom.com.br/uploads/product/file/a4d6b18b9b0cc9b4ac5dccfa22678fdd.pdf), this is the correct definition